content/blog/a-month-after-eclipsecon-adoptium-community-day-summary-and-more/index.md

```diff
@@ -0,0 +1,81 @@
+ - **Increase Product Usage**. Keep up with trends, and ensure we build what people need. Explain the real value of our products. We already have the “we produce high-quality binaries”. We want to show how these binaries are produced, talk with organizations, and ensure they understand the project's position.
+ - **Innovation that Matters**. Cover how we produce the projects, we can do a far better job and be more dynamic, as we have a big infrastructure to maintain. We want to generate collaborations with other Eclipse and influential projects beyond Eclipse.
+ - **Growing membership and involvement**. Participatory community, join the WG and bring your point of view. We need the voice of Java users and enterprises. It helps to bring additional funds and new opportunities, ensuring we have involvement from a wide variety of industries, have a robust community. 
+
+### Working Group and Community update. 
+Here we showed some ideas and opened a discussion on how we can promote community, working group, and sponsorship:
+ - **New members**. We want to hear about Enterprise pain points and be sure how we add value to them. Reach out to industry decision-makers by doing B2B Marketing 
+ - **Student programs**. We have collaborated with universities in the past. We are running the Outreachy program with two spaces: one funded by Eclipse and one by Red Hat.
+ - **More Events hosted by Adoptium**.  Virtual conferences and being an active part of existing Java community events. This is a line of direct sponsorship as well. 
+ - **Adopter page**. We could have an adopter page on our website for all users. It doesn’t matter if they belong to Eclipse Adoptium WG. 
+
+### Adoptium website
+Hosted and maintained by the community and Eclipse Foundation team. It was developed in a very open way and translated into various languages. For 2023, we want to ensure that it helps us achieve our focus areas and promotes our initiatives very easily, according to the audience profile. 
+
+### Adoptium Infrastructure
+The current infrastructure is composed of  300+ machines. The team is working on Infrastructure as code, as we rely on vendor collaborations and sponsorship. In this area, we need to be leaner and more efficient. The highest usage of machines is during release time, which happens every three months.  Our technical project management team is looking for collaborations in this area, as Adoptium is always looking to adopt best practices in the industry. 
+
+### Reproducible builds
+An important part of the software supply chain security. Currently, Temurin is compliant with SLSA level 2, and as part of the 2023 plan, we want to achieve level 4. This includes a number of requirements including having SBOM implemented (Software bill of materials) and forms part of Adoptium's [secure software development](https://adoptium.net/en-GB/docs/secure-software/) initiative.
```

Comment:
@tellison just checking on this statement. Is achieving SLSA Level 4 part of the 2023 program plan?

Reply:
Yes, it is part of the project priorities under "Secure Supply Chain and Software Engineering" topic.